tests/subsys/cpp/cxx/src/main.cpp

```diff
@@ -104,18 +104,10 @@ static int test_init(const struct device *dev)
 SYS_INIT(test_init, APPLICATION, CONFIG_APPLICATION_INIT_PRIORITY);
 
 
-static void test_new_delete(void)
+ZTEST(cpp_tests, test_new_delete)
 {
 	foo_class *test_foo = new foo_class(10);
 	zassert_equal(test_foo->get_foo(), 10, NULL);
 	delete test_foo;
 }
-
-void test_main(void)
-{
-	ztest_test_suite(cpp_tests,
-			 ztest_unit_test(test_new_delete)
-		);
-
-	ztest_run_test_suite(cpp_tests);
-}
+ZTEST_SUITE(cpp_tests, NULL, test_init, NULL, NULL. NULL);
```

Comment:
mistype "NULL. NULL"

Reply:
test_init is already executed by SYS_INIT(test_init, APPLICATION, CONFIG_APPLICATION_INIT_PRIORITY)

Reply:
ok, I will commit it again. thank you.